### PR TITLE
[FritzboxTr064] Fixed some examples in FritzBox TR064-Readme

### PR DESCRIPTION
**Pull request by Hunv**
Adjusted presence.map content to real values
Changed <present> icon to <presence>
Added remark to install Map Transformation Addon for using mapping of presence.map
Fixed variable in item configuration

### CODE REVIEW

**Comment by 9037568** [comment]
Please explain these changes.

**Comment by Hunv** [comment]
I just installed this binding and recognized that the documentation is not correct.
1. The real values of the call state are "OPEN", "CLOSED" and "NULL" (before the first status change)
2. I didn't know about the Map Transformation plugin, so I struggled with the "MAP"-stuff a while. The hint will save some time for other newbees
3. I am not really aware why and what the difference is, but the %d is not working while %s is working in the item definition
4. the Iconname is "presence" not "present"

**Comment by 9037568** [comment]
Do @tensberg, @gitbock, or @staehler have any comments regarding this proposed change?

**Comment by tensberg** [comment]
I never tried this out myself until today. The config works nicely with the corrections Hunv has made. Thanks @Hunv!

Before this is merged, the 1=present / 0=not present can be removed from the README, it's obsolete.

FYI Hunv regarding 3: %d is for decimal numbers, while %s is for strings, this is why %s is the right one for the real values.

File: bundles/binding/org.openhab.binding.fritzboxtr064/README.md [hunk 1]
```diff
@@ -120,12 +120,15 @@ Create file `transform/presence.map` and add:
 ```
 1=present
```
Comment (tensberg) [inline bundles/binding/org.openhab.binding.fritzboxtr064/README.md]: I thisnk you should remove the 1 and 0 lines while you're at it, these values are not used.

**Comment by Hunv** [comment]
Hi, I updated the file and removed the old 0 and 1 for the presence.map-file.

**Comment by 9037568** [comment]
Thanks @Hunv !

**Pull request merged**